plugins/json/core/src/test/java/org/pentaho/di/trans/steps/jsoninput/analyzer/JsonInputAnalyzerTest.java

```diff
@@ -196,4 +196,10 @@ public void testJsonInputExternalResourceConsumer() throws Exception {
 
```

Comment:
The test classes can extend  ClonableStepAnalyzerTest
```
public class JsonInputAnalyzerTest.java  extends ClonableStepAnalyzerTest
```

with overridding the method `newInstance`, you'll get for free the unit test `testCloneAnalyzer` which that each new call returns a unique instance.

Reply:
don't think this is valid due to it's not in the metaverse project.